src/Controls/src/Core/PublicAPI/net6.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,6957 @@
+~Microsoft.Maui.Controls.Compatibility.Grid.IGridList<T>.Add(Microsoft.Maui.Controls.View view, int left, int top) -> void
+~Microsoft.Maui.Controls.Compatibility.Grid.IGridList<T>.AddHorizontal(Microsoft.Maui.Controls.View view) -> void
+~Microsoft.Maui.Controls.Compatibility.Grid.IGridList<T>.AddHorizontal(System.Collections.Generic.IEnumerable<Microsoft.Maui.Controls.View> views) -> void
+~Microsoft.Maui.Controls.Compatibility.Grid.IGridList<T>.AddVertical(Microsoft.Maui.Controls.View view) -> void
+~Microsoft.Maui.Controls.Compatibility.Grid.IGridList<T>.AddVertical(System.Collections.Generic.IEnumerable<Microsoft.Maui.Controls.View> views) -> void
+~Microsoft.Maui.Controls.Compatibility.Grid.On<T>() -> Microsoft.Maui.Controls.IPlatformElementConfiguration<T, Microsoft.Maui.Controls.Compatibility.Grid>
+~Microsoft.Maui.Controls.Compatibility.Grid.RowDefinitions.get -> Microsoft.Maui.Controls.RowDefinitionCollection
+~Microsoft.Maui.Controls.Compatibility.Grid.RowDefinitions.set -> void
+~Microsoft.Maui.Controls.Compatibility.INativeElementView.Element.get -> Microsoft.Maui.Controls.Element
+~Microsoft.Maui.Controls.Compatibility.Layout.Children.get -> System.Collections.Generic.IReadOnlyList<Microsoft.Maui.Controls.Element>
+~Microsoft.Maui.Controls.Compatibility.Layout.LowerChild(Microsoft.Maui.Controls.View view) -> void
+~Microsoft.Maui.Controls.Compatibility.Layout.OnChildMeasureInvalidated(object sender, System.EventArgs e) -> void
+~Microsoft.Maui.Controls.Compatibility.Layout.RaiseChild(Microsoft.Maui.Controls.View view) -> void
+~Microsoft.Maui.Controls.Compatibility.Layout<T>
+~Microsoft.Maui.Controls.Compatibility.Layout<T>.Children.get -> System.Collections.Generic.IList<T>
+~Microsoft.Maui.Controls.Compatibility.Layout<T>.LayoutHandler.get -> Microsoft.Maui.ILayoutHandler
+~Microsoft.Maui.Controls.Compatibility.Layout<T>.this[int index].get -> Microsoft.Maui.IView
+~Microsoft.Maui.Controls.Compatibility.Layout<T>.this[int index].set -> void
+~Microsoft.Maui.Controls.Compatibility.RelativeLayout.Children.get -> Microsoft.Maui.Controls.Compatibility.RelativeLayout.IRelativeList<Microsoft.Maui.Controls.View>
+~Microsoft.Maui.Controls.Compatibility.RelativeLayout.IRelativeList<T>
+~Microsoft.Maui.Controls.Compatibility.RelativeLayout.IRelativeList<T>.Add(T view, Microsoft.Maui.Controls.Compatibility.Constraint xConstraint = null, Microsoft.Maui.Controls.Compatibility.Constraint yConstraint = null, Microsoft.Maui.Controls.Compatibility.Constraint widthConstraint = null, Microsoft.Maui.Controls.Compatibility.Constraint heightConstraint = null) -> void
+~Microsoft.Maui.Controls.Compatibility.RelativeLayout.IRelativeList<T>.Add(T view, System.Linq.Expressions.Expression<System.Func<double>> x = null, System.Linq.Expressions.Expression<System.Func<double>> y = null, System.Linq.Expressions.Expression<System.Func<double>> width = null, System.Linq.Expressions.Expression<System.F
```

Comment:
Would be nice if async APIs could be made consistent by adding an `Async` suffix everywhere.
See also https://github.com/dotnet/maui/issues/6720 (There are more cases than just these.)

Reply:
Not sure we can change APIs at this point. Maybe we can add an overload, but for a name-only thing?